src/node.cc

```diff
@@ -2220,6 +2221,40 @@ void Hrtime(const FunctionCallbackInfo<Value>& args) {
   fields[2] = t % NANOS_PER_SEC;
 }
 
+// Microseconds in a second, used in CPUUsage() below
+#define MICROS_PER_SEC 1000000
```

Comment:
Could change this to `#define MICROS_PER_SEC 1e6` then below do `MICROS_PER_SEC * rusage.*`. So you can drop the `1.0L`. Just a suggestion, not a blocker.